.travis.yml

```diff
@@ -33,22 +33,22 @@ matrix:
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: '7.0'
       env: SYMFONY=2.3.*
-    - php: '7.2'
-      env: SYMFONY=2.7.*
```

Comment:
Do not remove those 2 lines please

Reply:
Sorry, I thought that 2.7 branch of Symfony was not maintained anymore. I was wrong.